yarn.lock

```diff
@@ -2945,7 +2945,7 @@
     semver "^7.3.2"
     tsutils "^3.17.1"
 
-"@typescript-eslint/eslint-plugin@^4.4.1":
+"@typescript-eslint/eslint-plugin@^4.22.1":
```

Comment:
eslint failing on `git commit` for some reason. I `rm`d `node_modules` and did a clean install